packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -29,45 +24,49 @@ import { ContainerRuntime, IContainerRuntimeOptions } from "@fluidframework/cont
 import { IContainer } from "@fluidframework/container-definitions";
 import { Loader } from "@fluidframework/container-loader";
 
-class TestDataObject extends DataObject {
-	public get _root() {
-		return this.root;
-	}
+function getIdCompressor(dds: SharedObjectCore): IIdCompressor {
+	return (dds as any).runtime.idCompressor as IIdCompressor;
+}
 
-	public get _context() {
-		return this.context;
-	}
+describeNoCompat("Runtime IdCompressor", (getTestObjectProvider, apis) => {
+	const {
+		dataRuntime: { DataObject, DataObjectFactory },
+		containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
+	} = apis;
+	class TestDataObject extends DataObject {
+		public get _root() {
+			return this.root;
+		}
 
-	private readonly sharedMapKey = "map";
-	public map!: SharedMap;
+		public get _context() {
+			return this.context;
+		}
 
-	private readonly sharedCellKey = "sharedCell";
-	public sharedCell!: SharedCell;
+		private readonly sharedMapKey = "map";
+		public map!: SharedMap;
 
-	protected async initializingFirstTime() {
-		const sharedMap = SharedMap.create(this.runtime);
-		this.root.set(this.sharedMapKey, sharedMap.handle);
+		private readonly sharedCellKey = "sharedCell";
+		public sharedCell!: SharedCell;
 
-		const sharedCell = SharedCell.create(this.runtime);
-		this.root.set(this.sharedCellKey, sharedCell.handle);
-	}
+		protected async initializingFirstTime() {
+			const sharedMap = SharedMap.create(this.runtime);
```

Comment:
Shouldn't this be apis.SharedMap.create? Same for other DDSes.

Reply:
yes. good catch--thanks!